docs/book/src/plugins/available-plugins.md

```diff
@@ -0,0 +1,25 @@
+# Available plugins
+
+This section describes the plugins supported and shipped in with the Kubebuilder project see: 
+- [go/v3](go-v3-plugin.md)
+- [go/v2](go-v2-plugin.md)
```

Comment:
I think we need to have the plugins as sub-sections.

Note that this section will describe any plugin that is shipped by default via KB. (no only theese ones)